v1.1/parallel-statement-execution.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Parallel Statement Execution
+summary: The parallel statement execution feature allows parallel execution of multiple independent SQL statements within a transaction.
+toc: false
+---
+
+CockroachDB supports parallel execution of independent [`INSERT`](insert.html), [`UPDATE`](update.html), [`UPSERT`](upsert.html), and [`DELETE`](delete.html) statements within a single transaction. Parallel execution of SQL statements reduces the overall execution time of multiple SQL statements.
+
+<div id="toc"></div>
+
+## Why Use Parallel Statement Execution
+
+Normally, applications issue multiple SQL statements within a single transaction and SQL engines execute each statement sequentially. Because the statements are executed locally, the results can be delivered fast. But CockroachDB being a distributed database, execution of SQL statements takes longer than normal SQL engines because of high round-trip latencies required for consensus. To counteract the issue and optimize performance, CockroachDB supports parallel execution of SQL statements. Thus, despite the higher latency of single data-modifying statements, the overall execution latency of multiple statements can be reduced.
+
+## How Parallel Statement Execution Works
+
+SQL statements within a single transaction can be executed in parallel if the statements are independent. CockroachDB considers SQL statements within a single transaction to be independent if they operate on independent tables and their execution can be safely reordered without affecting their results. 
+
+Normal execution of SQL statements returns a successful return value immediately from the server. After the return value is received, the next SQL statement is executed. Appending the `RETURNING NOTHING` clause with SQL statements prevents the server from sending return values and allows CockroachDB to execute the statements in parallel. Thus to execute statements in parallel, append the `RETURNING NOTHING` suffix to the SQL statements. 
```

Comment:
Agreed. Will delete "successful"
Also, rewording the why use parallel statement section. Does seem better for the doc:

## Why use parallel statement execution
To maintain strong consistency across a globally-distributed cluster, CockroachDB replicates writes using the [consensus protocol](https://www.cockroachlabs.com/blog/consensus-made-thrive/). The protocol requires majority of replicas to agree on a write before committing it. Thus writes to CockroachDB require coordination between globally-distributed nodes. Communication between these nodes takes time and contributes to communication latency. Executing SQL statements sequentially, as in traditional SQL engines, leads to higher cumulative communication latency.

Parallel execution of statements can help reduce the communication latency. With parallel statement execution, multiple SQL statements within a transaction are executed at the same time, thereby reducing the aggregate latency over multiple SQL statements.